src/Fluid/Flu_ResetByUser.cpp

```diff
@@ -132,19 +136,19 @@ void Flu_ResetByUser_API_Default( const int lv, const int FluSg, const double Ti
 
          for (int v=0; v<NCOMP_TOTAL; v++)   fluid[v] = amr->patch[FluSg][lv][PID]->fluid[v][k][j][i];
 
+#        ifdef MHD
```

Comment:
Just notice that we implicitly assume users will enable `MHD` only in `MODEL = HYDRO`, and ignore the check `MODEL = HYDRO` as the code here. However, I do not find any check in `Aux_Check_Parameter.cpp` for that. Should we add it?

Reply:
@hfhsieh I don't think it's necessary since one can only enable `MHD` when adopting `MODEL=HYDRO` in `Makefile` ([here](https://github.com/gamer-project/gamer/blob/master/src/Makefile#L53)).